lib/onebox/engine/whitelisted_generic_onebox.rb

```diff
@@ -302,6 +308,17 @@ def is_embedded?
         )
       end
 
+      def card_html
+        <<~RAW
+        <iframe src="#{data[:player]}"
```

Comment:
We should probably ensure the URL is normalized by calling `::Onebox::Helpers.normalize_url_for_output`